packages/coil-extension/test.sh

```diff
@@ -36,7 +36,7 @@ function retry() {
 if [[ ${DEV} = 'false' ]]; then
   COMMAND="ts-node -r tsconfig-paths/register -T -P test/tsconfig.json"
 else
-  COMMAND="ts-node-dev -r tsconfig-paths/register -P test/tsconfig.json --respawn --transpile-only"
+  COMMAND="nodemon -r tsconfig-paths/register -P test/tsconfig.json --respawn --transpile-only"
```

Comment:
did you test this command works ?

Reply:
those options seem particular to `ts-node-x`

Reply:
![image](https://user-images.githubusercontent.com/525211/171090588-a5d82410-83aa-47df-9536-160422bcc568.png)